### PR TITLE
📖 update releasing.md

### PR DESCRIPTION
**Pull request by tuminoid**
With the learnings of minor release 0.4.0, update documentation.

### CODE REVIEW

File: docs/releasing.md [hunk 1]
```diff
@@ -70,7 +84,9 @@ We also need to create one or more tags for the Go modules ecosystem:
   `git tag -s api/v0.x.y -m api/v0.x.y`.
```
Comment (smoshiur1237) [inline docs/releasing.md:84]:
```suggestion
  `git tag -s apis/v0.x.y -m apis/v0.x.y`.
```

File: docs/releasing.md [hunk 2]
```diff
@@ -70,7 +84,9 @@ We also need to create one or more tags for the Go modules ecosystem:
...
   For BMO, these directories are `api` and `pkg/hardwareutils`. This enables
```
Comment (smoshiur1237) [inline docs/releasing.md:85]:
```suggestion
  For BMO, these directories are `apis` and `pkg/hardwareutils`. This enables
```

File: docs/releasing.md (hunk 1 above)
Comment (tuminoid) [reply docs/releasing.md]: fixed

File: docs/releasing.md (hunk 2 above)
Comment (tuminoid) [reply docs/releasing.md]: fixed

**Review by Sunnatillo — COMMENTED** [review]
LGTM

**Review by kashifest — COMMENTED** [review]
/approve

**Review by lentzi90 — COMMENTED** [review]
/lgtm

**Comment by tuminoid** [comment]
/override test-centos-e2e-integration-main
/override test-ubuntu-integration-main

Tests do nothing for readmes, skip.

**Pull request merged**